docs/user/registering-microshift-devices-acm.md

```diff
@@ -57,18 +58,20 @@ spec:
       os:
         image: quay.io/myorg/device-image-with-microshift:v1
       config:
-      - configType: HttpConfigProviderSpec
+      - name: acm-crd
         httpRef:
           filePath: /var/local/crd.yaml
```

Comment:
Could we write that to a subdirectory below `/var/local` instead, so we keep this system directory clean?

Reply:
Changed to /var/local/acm-import/